stl/inc/system_error

```diff
@@ -473,6 +473,8 @@ private:
     }
 
 protected:
+    _System_error(error_code _Errcode) : runtime_error(_Errcode.message()), _Mycode(_Errcode) {}

```

Comment:
Maybe we should add `explicit` to the unary constructor to prevent (internal) misuse, even though `system_error` constructors are not `explicit`?

Reply:
I wouldn't object to that, but I don't think there's a strong need for it, as this type isn't widely used and doesn't appear as a parameter anywhere.